src/udiskslinuxencrypted.c

```diff
@@ -516,6 +520,14 @@ handle_unlock (UDisksEncrypted        *encrypted,
   else {
     if (is_luks)
       name = g_strdup_printf ("luks-%s", udisks_block_get_id_uuid (block));
+    else if (is_bitlk)
+      {
+        uuid = udisks_block_get_id_uuid (block);
```

Comment:
Currently UUID is not set for BitLocker devices because bitlk doesn't support it but this might change in the future.